src/harness/sas_testcase.py

```diff
@@ -96,13 +96,42 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
       self.assertEqual(grant_resp['response']['responseCode'], 0)
 
       list_of_cbsd_ids_and_grant_ids.append({
-          'cbsdId': grant_resp['cbsdId'],
-          'grantId': grant_resp['grantId']
+        'cbsdId': grant_resp['cbsdId'],
+        'grantId': grant_resp['grantId']
       })
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
 
+  def assertHeartbeatsSuccessful(self, cbsd_ids, grant_ids, operation_states):
```

Comment:
If I am correct then I have to change the input of this function as well? Instead of taking cbsd_ids and grant_ids separately, do you want me to take this as a tuple as well?